openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/shared/DiffViewNew.tsx

```diff
@@ -36,69 +37,80 @@ export const DiffViewNew = ({
   const [shouldShowViewMore, setShouldShowViewMore] = useState<boolean>(false);
   const contentRef = useRef<HTMLDivElement>(null);
 
-  function stripHtml(html: string) {
-    return html
-      .replace(/<(?!\/?strong\b)[^>]+>/g, '')
-      .replace(/\*\*(.*?)\*\*/g, '<strong>$1</strong>')
-      .trim();
-  }
+  useEffect(() => {
+    const checkHeight = () => {
+      if (contentRef.current) {
+        const lineHeight = parseInt(
+          window.getComputedStyle(contentRef.current).lineHeight,
+          10
+        );
+        const twoLinesHeight = lineHeight * 2;
+
+        // Get the actual content height without restrictions
+        const clone = contentRef.current.cloneNode(true) as HTMLElement;
+        clone.style.maxHeight = 'none';
+        clone.style.position = 'absolute';
+        clone.style.visibility = 'hidden';
+        document.body.appendChild(clone);
+        const fullHeight = clone.scrollHeight;
+        document.body.removeChild(clone);
+
+        // Compare the full height with two lines height
+        setShouldShowViewMore(fullHeight > twoLinesHeight);
```

Comment:
Not sure why we want this logic

Reply:
To check the ht of the lines being displayed, since we are showing the view more btn only after two lines are being rendered. Earlier in our legacy code , we were showing the view more based on the length of  characters being rendered. Now in the updated designs we are showing it after two lines . This logic basically is creating an invisible copy of an element to measure its true height .